.github/workflows/CodeQuality.yml

```diff
@@ -14,7 +14,6 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  GH_TOKEN: ${{ secrets.GH_TOKEN }}
   TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
   AWS_ACCESS_KEY_ID: ${{ secrets.S3_ID }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.NODE_PRE_GYP_SECRETACCESSKEY }}
```

Comment:
hey @carlopi looks good! i think one thing we also really want here is to move all of the secrets down to steps where we use them. We can have a look at this together tomorrow?

Reply:
sure!
here it was mostly a trial-run on one of them, to check whether this all makes sense. Will go over the other ones then, and tomorrow we fill the gaps.